src/main/java/Database/artists.csv

```diff
@@ -0,0 +1,2 @@
+Artist1,Password1,Artist Name 1,110 200 3
```

Comment:
This is optional but I feel like separating the song IDs with commas would be nice (a lot of other data is going to end up being formatted that way, it'd be nice to reuse code but we could always just make it so that it takes in what to splice by).

Reply:
While I don't disagree that it looks nicer, since I am using .csv files (comma separated values), that would add columns and break the parsing. 

Also, I've already written the parsing methods for these databases, just haven't pushed yet since they rely on dependencies that don't exist as of today.

Reply:
Wow I completely forgot about the comma part of 'comma separated values' -- my bad, I probably shouldn't be reviewing this at 12am lol. Sticking to semicolons would be nice in case the difference causes some issue later down the line but it seems unlikely

Reply:
Yep, honestly I should've been consistent with my sub-delimiters anyways. I'll change the spaces to semicolons tomorrow and re-request your review.